homeassistant/bootstrap.py

```diff
@@ -252,8 +252,7 @@ async def async_from_config_dict(
             f"{'.'.join(str(x) for x in sys.version_info[:3])} is deprecated and will "
             f"be removed in Home Assistant {REQUIRED_NEXT_PYTHON_HA_RELEASE}. "
             "Please upgrade Python to "
-            f"{'.'.join(str(x) for x in REQUIRED_NEXT_PYTHON_VER)} or "
-            "higher."
+            f"{'.'.join(str(x) for x in REQUIRED_NEXT_PYTHON_VER)}."
```

Comment:
Although better, it's also not correct because we don't just want 3.9.0, it is also ok to have 3.9.9

Reply:
Right. Just iterating over the first two components of the version tuple and showing "Please upgrade to Python 3.9"?